lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -176,10 +180,16 @@ def test_section_is_none(self):
             section_url,
             r'courseware/test_factory_chapter_omega_%CE%A9/$'
         )
-        self.assertTrue(section_url)
-        self.assertTrue(unit_url)
+
         self.assertEqual(section_url, unit_url)
 
+        path = course_outline[0]['path']
+        self.assertEqual(1, len(path))
+        #Named path will be deprecated eventually
+        named_path = course_outline[0]['named_path']
+        self.assertEqual(1, len(named_path))
+        self.assertEqual(u'test factory chapter omega \u03a9', named_path[0])
```

Comment:
Why not verify the content of `path` in these tests instead of `named_path`?  Especially since `named_path` is being deprecated and not used by the apps?